content/2016/09/30/cssconf.rst

```diff
@@ -0,0 +1,640 @@
+
+`Henri Helvetica`_
+talked about optimizing page and image sizes
+for the web.
+Did you know mp4 videos
+have better performance than gif images?
+Sites like Twitter
+convert your animated gif into mp4 format
+for display.
+
+.. _Henri Helvetica: http://twitter.com/HenriHelvetica
+
+
+Sass Map Magic
+--------------
+
+.. image:: /static/images/blog/cssconf-2016/mia.jpg
+
+I showed a wide range of uses
+for the underused Sass "map" data type —
+from simple site theme configurations,
+to data storage,
+and complex functional programming.
+All the
+`slides <https://oddbooksapp.com/book/sass-map-magic>`_
+are online.
+
+.. _Miriam Suzanne: http://twitter.com/mirisuzanne
+
+
+Webpack and CSS
+---------------
+
+.. image:: /static/images/blog/cssconf-2016/zg.jpg
+
+`Zach Green`_
+walked us through his Webpack setup.
+I missed most of this,
+recovering from my own talk.
+
+.. _Zach Green: http://twitter.com/zgreen_
+
+
+It's Time To Ditch The Grid System
+----------------------------------
+
+.. image:: /static/images/blog/cssconf-2016/eh.jpg
+
+`Emily Hayman`_
+demonstrated the ins and outs
+of using flexbox to build
+"content-driven" layouts,
+instead of forcing our content into grid colums.
+It's a great overview,
+and I particularly resonate with the
+"step lightly" philosophy
+that was repeated here.
+If you need a refresher
+on the *how* and *why* of flexbox,
+this is a great place to start.
+
+.. _Emily Hayman: http://twitter.com/eehayman
+
+
+Bauhaus in the Browser
+----------------------
+
+.. image:: /static/images/blog/cssconf-2016/jm.jpg
+
+`Justin McDowell`_
+used CSS transforms, grids, and more
+to bring bauhaus-inspired art and layouts
+to the browser.
+It's a fun and beautiful talk,
+that includes a demonstration of
+"Dolly zoom"
+(also known as the "vertigo effect")
+in CSS.
+
+.. _Justin McDowell: http://twitter.com/revoltpuppy
+
+
+The Great SVG RetCon
+--------------------
+
+.. image:: /static/images/blog/cssconf-2016/ab.jpg
+
+`Amelia Bellamy-Royds`_
+gave us a full overview of changes
+in SVG2,
+along with a history of SVG.
+This talk is packed full of useful information,
+if you are using SVG in any way.
+
+.. _`Amelia Bellamy-Royds`: http://twitter.com/AmeliasBrain
+
+
+Coding is a Privilege
+---------------------
+
+.. image:: /static/images/blog/cssconf-2016/ar.jpg
+
+`Alisha Ramos`_
+closed out the conference
+with a rousing talk about diversity
+(and privilege!)
+in tech.
+Key take-aways:
+
+- It's important to be aware
+  of the privileges
+  that got you where you are.
+- Diversity is not *just* a pipeline issue.
+  Representation is worse in the workforce
+  than it is in training programs.
+  A pipeline is only as useful as the place it takes you.
+- Culture-fit can be problematic
+  when it refers to "drinking buddies"
+  instead of shared values.
+
+I would have taken better notes,
+but I was too busy applauding.
+This was a great way to end the conference.
+You should watch the video.
```

Comment:
Maybe link "watch the video" to the video: https://www.youtube.com/watch?v=PtKOzKNJF-s